src/testing/loadtest-azure/scripts/file-upload-to-loadtest.ps1

```diff
@@ -0,0 +1,47 @@
+param
+(
+  # Load Test Id
+  [Parameter(Mandatory = $true)]
+  [string] $loadTestId,
+
+  # Load Test data plane endpoint
+  [Parameter(Mandatory = $true)]
+  [string] $apiEndpoint,
+
+  # Load Test data plane api version
+  [string] $apiVersion = "2021-07-01-preview",
```

Comment:
I know you just moved those into the scripts but I'm not sure this is easier to update? and as this is still preview, I would expect a few more api updates to come

Reply:
ok, an we then also remove the API version in the central config file alltogether?

Reply:
Done